templates/humanagent/src/skills/waas.ts

```
@@ -0,0 +1,91 @@
+  switch (skill) {
+    case "create":
+      const created = await walletService.createWallet(sender.address);
+      if (created) {
+        await context.reply(
+          "Your CDP wallet has been created! You can now fund it using /fund command.",
+        );
+      }
+      break;
+
+    case "fund":
+      const { amount } = params;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)